eng/Versions.props

```diff
@@ -15,7 +15,7 @@
   <PropertyGroup>
     <MajorVersion>3</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>2</PatchVersion>
+    <PatchVersion>3</PatchVersion>
     <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
```

Comment:
@mmitche I want to double check if we should be flipping the StabilizePackageVersion property back to false?

Reply:
Good question

Reply:
I don't think we do, as long as this is the first thing to go in.

Reply:
Okay, I'll leave these as is then.